src/Slider/Slideshow.js

```diff
@@ -35,17 +38,35 @@ const SlideshowDot = styled.div`
   background-color: ${(props) => (props.active ? "#303030" : "#c4c4c4")};
 `;
 
+const SlideText = styled.div`
+  position: absolute;
+  bottom: 10px;
+  right: 10px;
+  text-align: right;
+  color: white;
+  text-shadow: 2px 2px 2px #000000;
+  font-size: 1.2em;
+`;
+
 function Slideshow({ slides }) {
   const [index, setIndex] = React.useState(0);
 
   return (
     <SlideshowContainer>
       <SlideshowSlider index={index}>
-        {slides.map((slide, index) => (
-          <div key={index}>
-            <img src={slide.url} alt={slide.alt} />
-          </div>
-        ))}
+        {slides.map((slide, index) =>
+          slide.navigateTo === undefined ? (
```

Comment:
Esta línea se puede remplazar por 
```suggestion
          !slide.navigateTo ?
```

Se evalua igual